enterprise/app/history/history.tsx

```diff
@@ -280,6 +286,10 @@ export default class HistoryComponent extends React.Component<Props, State> {
       selectedInvocationIndex: newInvocationIndex,
       selectedInvocationId: newInvocationId,
     });
+    const elements = document.getElementsByClassName("selected-keyboard-shortcuts");
+    if (elements.length == 1) {
+      (elements[0] as HTMLElement).focus();
```

Comment:
not a huge deal but I think this isn't actually focusing the element since `<div>` isn't focusable by default. Since `<a>` is focusable you could focus the parent `<a>` element like `document.querySelector('a:has(.selected-keyboard-shortcuts)')?.focus();`

Reply:
https://github.com/buildbuddy-io/buildbuddy/assets/2414826/30acd145-b21d-4a00-a1cb-04c22a8ae67a

Reply:
Fixed